tidb-cloud/connect-via-standard-connection-serverless.md

```diff
@@ -34,6 +36,28 @@ To connect to a TiDB Serverless cluster via public endpoint, take the following
     >
     > When you connect to a TiDB Serverless cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).
 
+## Disable a public endpoint
+
+If you do not need to use a public endpoint of a TiDB Serverless cluster, you can disable it to prevent connection from the Internet:
```

Comment:
```suggestion
If you do not need to use a public endpoint of a TiDB Serverless cluster, you can disable it to prevent connections from the internet:
```